interfaces/tracing/v0/schemas/provider.json

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://canonical.github.io/charm-relation-interfaces/interfaces/tracing/schemas/provider.json",
+    "type": "object",
+    "title": "`tracing` provider schema",
+    "description": "The `tracing` root schema comprises the entire provider databag for this interface.",
+    "default": {},
+    "examples": [
+        {
+            "application-data": {
+                "hostname": "tempo.example.org",
+                "ingester-ports": [
+                    {
+                        "type": "otlp_grpc",
+                        "port": 1234
+                    },
+                    {
+                        "type": "otlp_http",
+                        "port": 1233
+                    }
+                ]
+            },
+            "related-units": {
```

Comment:
In most json schema specs here we don't include the generic data that we'll find in all interfaces like `related_units`